wifiphisher/common/phishingpage.py

```diff
@@ -1,19 +1,14 @@
-"""
-This module handles all the phishing related operations for
-Wifiphisher.py
-"""
+"""This module handles all the phishing related operations."""
 
+from __future__ import (absolute_import, division, print_function)
 import os
 import ConfigParser
 from shutil import copyfile
 import wifiphisher.common.constants as constants
 
 
 def config_section_map(config_file, section):
-    """
-    Map the values of a config file to a dictionary.
-    """
-
+    """Map the values of a config file to a dictionary."""
```

Comment:
s/Map/Maps

Reply:
From PEP257
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

Reply:
Interesting, good to know.

Maybe an idea is to gather all these interesting PEP rules on a page. We can name the page "Contributing to Wifiphisher" and we can refer to that page when we are doing code quality checks.